File: TP/TP1/Code/5F.cpp
/* Afficher le contour d'un rectangle de taille N * M */
#include <iostream>

int main()
{
  int n;
  int m;
  
  std::cout << "Quelle est la largeur du rectangle ?" << std::endl;
  std::cin >> n;
	std::cout << "Quelle est la hauteur du rectangle ?" << std::endl;
  std::cin >> m;
  
  for (int i = 0; i < m; ++i) {
    for (int j = 0; j < n; ++j) {
      if (i == 0 || i == m - 1 || j == 0 || j == n - 1)
        std::cout << "*" << std::flush;
      else
        std::cout << " " << std::flush;
    }
    std::cout << std::endl;
  }

  return 0;
}

File: TP/TP1/Code/Python/5F.py
#!/usr/bin/python
# Afficher le contour d'un rectangle de taille N * M
print('Donne moi la longueur : ', end='')
foo = int(input())
print('Donne moi la hauteur : ', end='')
bar = int(input())

for i in range(bar):
    for j in range(foo):
        if i == 0 or i == bar - 1:
            print('*', end='')
        else:
            if j == 0 or j == foo - 1:
                print('*', end='')
            else:
                print(' ', end='')
    print()  # Ici on affiche un saut de ligne
